crates/tuono/src/build.rs

```diff
@@ -0,0 +1,89 @@
+use console::Term;
+use fs_extra::dir::{copy, CopyOptions};
+use spinners::{Spinner, Spinners};
+use std::io;
+use std::path::PathBuf;
+use std::thread::sleep;
+use std::time::Duration;
+
+use crate::app::App;
+use crate::mode::Mode;
+
+#[tokio::main]
+pub async fn build(mut app: App, ssg: bool, no_js_emit: bool) -> Result<(), io::Error> {
```

Comment:
No need for `tokio::main`, `async` and a result type
```suggestion
pub fn build(mut app: App, ssg: bool, no_js_emit: bool) {
```